### PR TITLE
Gekidou bugs fixes

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
Fixes various bugs, if anything feel free to push a commit into this PR

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-45621
https://mattermost.atlassian.net/browse/MM-45604
https://mattermost.atlassian.net/browse/MM-45591
https://mattermost.atlassian.net/browse/MM-45590
https://mattermost.atlassian.net/browse/MM-45589
https://mattermost.atlassian.net/browse/MM-45572
https://mattermost.atlassian.net/browse/MM-45549
https://mattermost.atlassian.net/browse/MM-45720

```release-note
NONE
```

### CODE REVIEW

**Review by avinashlng1080 — APPROVED** [review]

File: app/screens/settings/display_timezone_select/index.tsx [hunk 1]
```diff
@@ -131,10 +138,9 @@ const SelectTimezones = ({selectedTimezone, onBack}: SelectTimezonesProps) => {
                 keyExtractor={keyExtractor}
                 keyboardDismissMode='on-drag'
                 keyboardShouldPersistTaps='always'
-                maxToRenderPerBatch={15}
                 removeClippedSubviews={true}
                 renderItem={renderItem}
-                viewabilityConfig={VIEWABILITY_CONFIG}
+                contentContainerStyle={{flexGrow: 1}}
```
Comment (larkox) [inline app/screens/settings/display_timezone_select/index.tsx:143]: Not blocking, but better to have it in the styles.

**Review by larkox — APPROVED** [review]
LGTM

File: app/screens/settings/display_timezone_select/index.tsx (hunk 1 above)
Comment (enahum) [reply app/screens/settings/display_timezone_select/index.tsx:143]: please make the change.

**Comment by enahum** [comment]
PR has two reviews, no need for a third

**Pull request merged**